cypress/support/commands.js

```diff
@@ -32,10 +32,33 @@ Cypress.Commands.add('getByDataCy', (selector) => {
   cy.get(`[data-cy="${selector}"]`);
 });
 
-Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
-  cy.request('POST', '/users', {
-    email,
-    username,
-    password
+Cypress.Commands.add('register',
+  (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+    cy.request('POST', '/users', {
+      email,
+      username,
+      password
+    });
+  });
+
+Cypress.Commands.add('login',
+  (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
+    cy.request('POST', '/api/users', {
```

Comment:
Ensure that the API endpoint `/api/users` is correct for the login operation. Typically, login endpoints might be something like `/api/login` or `/api/auth/login`.